models/userview.go

```diff
@@ -10,6 +10,7 @@ type UserView struct {
 	Id       uint   `json:"id,omitempty"`
 	Username string `json:"username,omitempty"`
 	Email    string `json:"email,omitempty"`
+	OtpActive bool `json:"otpactive"`
```

Comment:
Use pascalCasing for the JSON field, so otpActive

Reply:
Adjusted in [40f27bc](https://github.com/pufferpanel/pufferpanel/pull/1374/commits/40f27bc2575a019a3c7a3aaff54990215ea81c87)